Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/ui/validationresult/passed/DccValidationPassedViewModel.kt

```diff
@@ -42,7 +42,7 @@ class DccValidationPassedViewModel @AssistedInject constructor(
         return listOf(
             ValidationInputVH.Item(validation),
             ValidationOverallResultVH.Item(validation.state),
-            RuleHeaderVH.Item(type = validation.state, validation.rules.size),
+            RuleHeaderVH.Item(type = validation.state, ruleCount = validation.rules.size),
```

Comment:
can't we just pass `validation` and display what you is needed ? instead of de-structuring it in the args ?

Reply:
I don't think so. We could use the whole `DccValidation` object in `DccValidationPassedViewModel` but in other viewmodels for example `DccValidationOpenViewModel` we use state/result for individual rules. It's basically a header for a group of rules not for the whole fragment.

Reply:
Ok